app/docs/0.11.x/configuration.md

```diff
@@ -959,20 +959,6 @@ Default: `1`
 
 ---
 
-##### **lua_code_cache**
```

Comment:
since this is still present in 0.11.0, should we just leave a note that its been removed in >= 0.11.1?